index.html

```diff
@@ -18,26 +21,170 @@ <h2 class="log"><a class="Welcome" href="#">Welcome</a></h2>
         </nav>
     </header>
     <main>
-    <section class="section-1"> 
-        <h1 class="heading-1">Hey there. I’m Josphat</h1>
+        <section class="section-1">
+            <h1 class="heading-1">Hey there. I’m Josphat</h1>
             <h2 class="heading-2">I’m a software developer</h2>
-                <p class="par-1"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
-            don’t hestiate to
-                </p>
-        <ul class="icons">
-            <li><i class="fa fa-twitter" aria-hidden="true"></i></li>
-            <li><i class="fa fa-linkedin" aria-hidden="true"></i></li>
-            <li><i class="fa fa-medium" aria-hidden="true"></i></li>
-            <li><i class="fa fa-github" aria-hidden="true"></i></li>
-            <li><i class="fa fa-angellist" aria-hidden="true"></i></li>
-        </ul>
-    </section>
-</main>
+            <p class="par-1"> I can help you build a product , feature or website Look through some of my work and
+                experience! If you like what you see and have a project you need coded,
+                don’t hestiate to
+            </p>
+            <ul class="icons">
+                <li><i class="fa fa-twitter" aria-hidden="true"></i></li>
+                <li><i class="fa fa-linkedin" aria-hidden="true"></i></li>
+                <li><i class="fa fa-medium" aria-hidden="true"></i></li>
+                <li><i class="fa fa-github" aria-hidden="true"></i></li>
+                <li><i class="fa fa-angellist" aria-hidden="true"></i></li>
+            </ul>
+            <div class="down-img">
+                <img src="img/Disabled.png" class="disable" alt="disble">
+            </div>
+        </section>
+
+        <section class="work-section">
+            <div class="section-title">
+                <h2>My recent Works</h2>
+            </div>
+            <div class="green-line">
+                <img src="img/Indicator.png" alt="indicator">
```

Comment:
- [OPTIONAL] What the previous reviewer mean here is all decoration on the page should be background images as stated in the project requirement and not an image👍.  Kindly see to that😉